src/main/java/com/flightstats/hub/cluster/CuratorCluster.java

```diff
@@ -78,20 +82,19 @@ public void register() {
             log.info("this hub is read only, not registering");
             return;
         }
-        String host = Cluster.getHost(useName);
         try {
-            log.info("registering host {} {}", host, clusterPath);
-            curator.create().withMode(CreateMode.EPHEMERAL).forPath(getFullPath(), host.getBytes());
+            log.info("registering host {} {}", localHostProperties.getHost(useName), clusterPath);
```

Comment:
I think I'd've been tempted to pull a `getHost()` out into a private method since it's being used in this method 4 times and again in a few other methods.